### PR TITLE
Release 4.13 revendor

### PR DESCRIPTION
**Pull request by anshulvermapatel**
Openshift 4.13 update. Should be reviewed in conjunction with the forked installer branch: https://github.com/jewzaam/installer-aro/tree/release-4.13-azure

 bumped forked installer mod to 4.13-azure
 RHCOS updated to 4.13
 Enhancements to networking metrics
 Support for manifest listed images on imagestreams
 cgroup v2 general available
 crun general available

**Plan for testing: smoke test in dev, int and canary according to these steps**
An upgrade to the next Y version is possible
E2E tests are passing before and after the Y version upgrade
Public and private clusters can be created and both pass the smoke tests below
**Other smoke tests:**
Check ARO conditions with oc get cluster/cluster -o jsonpath='{.status.conditions}' | jq
Check ARO operator logs for errors with oc logs -n openshift-azure-operator -l app=aro-operator-master and oc logs -n openshift-azure-operator -l app=aro-operator-worker
Can run commands like oc new-app nginx && expose svc/nginx and the Route works from a browser (be sure to use http:// not https://)
Can provision/mount PVs created from all StorageClasses
Can scale MachineSets
Can create Services of type LoadBalancer
Check Node health
Check Cluster Operator health
PUCM and roll-out of new operator complete without errors in the RP and operator logs
az aro update succeeds in rotating certs and SP

**The above smoke tests are DONE.**

### CODE REVIEW

**Review by Shivkumar13 — APPROVED** [review]
We have tested image with these changes with the new installation and we can see bootstrap node is getting ahead successfully. We will be performing few more checks. Overall changes looks good to me. 

PTAL @bennerv @petrkotas @SudoBrendan @dofinn @s-amann

File: Makefile [hunk 1]
```diff
@@ -18,10 +18,8 @@ ifeq ($(RP_IMAGE_ACR),arointsvc)
 	REGISTRY = arointsvc.azurecr.io
 else ifeq ($(RP_IMAGE_ACR),arosvc)
 	REGISTRY = arosvc.azurecr.io
-else ifeq ($(RP_IMAGE_ACR),)
-	REGISTRY = registry.access.redhat.com
 else
-	REGISTRY = $(RP_IMAGE_ACR)
+	REGISTRY = registry.access.redhat.com
```
Comment (bennerv) [inline Makefile:22]: Why this change?  This drifts from our ARO-RP Makefile.  

Essentially if you have an override set, it will use that, otherwise it will default to registry.access.redhat.com

File: pkg/installer/generateconfig.go [hunk 2]
```diff
@@ -150,115 +150,117 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
...
-							DiskSizeGB:        1024,
+				SSHKey:     sshkey.Type() + " " + base64.StdEncoding.EncodeToString(sshkey.Marshal()),
+				BaseDomain: domain[strings.IndexByte(domain, '.')+1:],
+				Networking: &types.Networking{
+					MachineNetwork: []types.MachineNetworkEntry{
+						{
+							CIDR: *ipnet.MustParseCIDR("127.0.0.0/8"), // dummy
+						},
+					},
+					NetworkType: SoftwareDefinedNetwork,
```
Comment (bennerv) [inline pkg/installer/generateconfig.go:169]: Can we make this lowercase?  Otherwise the value is exported from the package and I don't think that's what we should be doing for future stuff.

Comment (bennerv) [reply pkg/installer/generateconfig.go:169]: `SoftwareDefinedNetwork`

File: pkg/installer/generateconfig.go [hunk 3]
```diff
@@ -150,115 +150,117 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
...
+					// don't include the baremetal capability (in the baseline default)
+					BaselineCapabilitySet: configv1.ClusterVersionCapabilitySetNone,
+					AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{
+						configv1.ClusterVersionCapabilityConsole,
+						configv1.ClusterVersionCapabilityCSISnapshot,
+						configv1.ClusterVersionCapabilityInsights,
+						configv1.ClusterVersionCapabilityMarketplace,
+						configv1.ClusterVersionCapabilityOpenShiftSamples,
+						configv1.ClusterVersionCapabilityStorage,
+					},
```
Comment (bennerv) [inline pkg/installer/generateconfig.go:263]: are there new capabilities in 4.13 that should be enabled as well?  

`NodeTuning`?

File: Makefile (hunk 1 above)
Comment (anshulvermapatel) [reply Makefile:22]: it contradicts with the line 4, if $(RP_IMAGE_ACR) is empty, then $ARO_IMAGE_BASE would be .azurecr.io/aroinstaller. At the same time we gave a condition where we are expecting $(RP_IMAGE_ACR) to be empty in the else if condition which I removed.
Also to build the aro-installer locally `ubi-minimal` image is used as the base image which is publicly available in the registry.access.redhat.com registry.

File: pkg/installer/generateconfig.go (hunk 2 above)
Comment (anshulvermapatel) [reply pkg/installer/generateconfig.go:169]: we can, it was as it previously, but yeah we can definitely make it lowercase now.

File: pkg/installer/generateconfig.go (hunk 3 above)
Comment (anshulvermapatel) [reply pkg/installer/generateconfig.go:263]: We can definitely, sorry that we missed it. I added to the list of enabled capabilities.

**Review by Shivkumar13 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (bennerv) [reply Makefile:22]: makes sense to me.  want to change this in ARO-RP as well then?

**Review by bennerv — APPROVED** [review]
How do we sync this to master?  Ideally everyone will work off of master + then cherry-pick to release branches.  That's my only concern here, otherwise LGTM.

File: Makefile (hunk 1 above)
Comment (Shivkumar13) [reply Makefile:22]: @bennerv 
Raised one here: https://github.com/Azure/ARO-RP/pull/3299
